CHANGES.md

```diff
@@ -225,6 +225,8 @@ To be released.
     it fails to connect to all neighbors.  [[#933]]
  -  `Swarm<T>` became to respond to the messages with different app protocol version.
     [[#949]]
+ -  `Swarm<T>.PreloadAsync()` became to execute the actions from the branch point
+    rather than the genesis block when there is a branch point.
```

Comment:
```suggestion
    rather than the genesis block when there is a branch point.  [[#991]]
```

And the link should be listed at line [L313](https://github.com/planetarium/libplanet/pull/991/files#diff-8b1c3fd0d4a6765c16dfd18509182f9dR313)

Reply:
I forgot about it. amended!